.github/pull_request_template.md

```diff
@@ -1,26 +1,17 @@
-## Pull Request Info
+# Pull Request Info
 
-[PR Reviewing Guidelines](https://github.com/mongodb/docs-java/blob/master/REVIEWING.md)
-
-### Issue JIRA link:
-https://jira.mongodb.org/browse/DOCSP-NNNNN
+[JIRA](https://jira.mongodb.org)
+[Staging](https://www.example.com)
 
-### Snooty build log:
-**Paste your workerpool job link here**
+## Reviewing
 
-### Docs staging link (requires sign-in on MongoDB Corp SSO):
-https://docs-mongodborg-staging.corp.mongodb.com/java/docsworker-xlarge/NNNNNNNNNNN/
+[PR Reviewing Guidelines](https://github.com/mongodb/docs-java/blob/master/REVIEWING.md)
```

Comment:
Suggestion: Leave this at the top as there's a better chance that reviewers will see and read it. Placing it in a following section positions it next to items that are less relevant to the reviewer.